co2_scd4x.ino

```diff
@@ -165,19 +248,37 @@ void goto_light_sleep(int ms){
 #ifdef WIFI
   delay(ms);
 #else
-  esp_sleep_enable_timer_wakeup(ms*1000);  // periodic measurement every 5 sec -1.1 sec awake
-  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON);    // RTC IO, sensors and ULP co-processor
-  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_SLOW_MEM, ESP_PD_OPTION_AUTO); // RTC slow memory: auto
-  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_FAST_MEM, ESP_PD_OPTION_OFF);  // RTC fast memory
-  esp_sleep_pd_config(ESP_PD_DOMAIN_XTAL, ESP_PD_OPTION_OFF);          // XTAL oscillator
-  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC8M, ESP_PD_OPTION_OFF);         // CPU core
+  esp_sleep_enable_timer_wakeup(ms*1000);                               // periodic measurement every 5 sec -1.1 sec awake
+  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_PERIPH, ESP_PD_OPTION_ON);      // RTC IO, sensors and ULP co-processor
+  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_SLOW_MEM, ESP_PD_OPTION_AUTO);  // RTC slow memory: auto
+  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC_FAST_MEM, ESP_PD_OPTION_OFF);   // RTC fast memory
+  esp_sleep_pd_config(ESP_PD_DOMAIN_XTAL, ESP_PD_OPTION_OFF);           // XTAL oscillator
+  esp_sleep_pd_config(ESP_PD_DOMAIN_RTC8M, ESP_PD_OPTION_OFF);          // CPU core
   esp_sleep_pd_config(ESP_PD_DOMAIN_VDDSDIO, ESP_PD_OPTION_OFF);
   esp_light_sleep_start();
 #endif
   //free(BlackImage);
   //BlackImage = NULL;
 }
 
+void updateBatteryMode(){
+  BatteryMode = (digitalRead(4) == LOW);
+}
+
+float readBatteryVoltage(){
+  // IO5 measurement was 5210 with 0.0 offset
+  return ((analogRead(5)*3.33)/5084) + 0.02;
+}
+
+uint8_t calcBatteryPercentage(float voltage) {
+  if (voltage <= 3.61)
```

Comment:
Den Algorithmus habe ich etwas angepasst (voltage += 0.11; // offset for type of Battery used)
https://github.com/davidkreidler/SCD4x_CO2_Sensor_ESP32/commit/87c00e7647bd17263353a508629dbab68f08d660

Reply:
Ist durch das rebase eingeflossen